renovate.json

```
@@ -126,6 +126,21 @@
       "matchBaseBranches": ["main"],
       "labels": ["Team:Security", "release_note:skip", "auto-backport"],
       "enabled": true
+    },
+    {
+      "groupName": "ftr",
+      "packageNames": [
+        "@types/chromedriver",
+        "@types/selenium-webdriver",
+        "chromedriver",
+        "geckodriver",
+        "ms-chromium-edge-driver",
+        "selenium-webdriver"
+      ],
+      "reviewers": ["team:kibana-operations"],
+      "matchBaseBranches": ["main"],
+      "labels": ["Team:Operations", "release_note:skip", "auto-backport", "v8.1.0", "v8.0.0", "v7.17.0", "v7.16.0"],
```

```suggestion
      "labels": ["Team:Operations", "release_note:skip"],
```

Let's not include the versions since we would need to update them on every bump. We can identify where it should be backported to at the time it's opened. Additionally, we should very rarely be backporting that far back.